findspam.py

```diff
@@ -18,7 +18,7 @@
 # noinspection PyPackageRequirements
 from tld.utils import TldDomainNotFound
 import phonenumbers
-import dns.resolver
+import dns.exception
```

Comment:
While this appears to work (it actually results in all of `dns` being pulled in, rather than *just* `dns.exception`), it seems like it would be better for this to be stated as `dns.resolver`, as that's what we're actually using.